Add requisition list to the supported REST import types (#565)
AdobeDocs/commerce-webapi: src/pages/rest/modules/import/index.md

|-------------------------------------|-----|-----|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| `locale`                            | The language and country combination for the data being imported | string | Optional                                                                                                                                                                                                                              |
| `entity`                            | The type of entity to be imported | string | Required. The value must be one of the following: `advanced_pricing`, `catalog_product`, `customer`, `customer_address`, `customer_composite`, `customer_finance`, `requisition_list` or `stock_sources`.                                                                                   |

> | `behavior`                          | The action to perform | string | Required. For `advanced_pricing`, `catalog_product`, `customer_composite`, `requisition_list`, or `stock_sources` the value must be one of the following: `append`, `replace`, or `delete`. For `customer`, `customer_address`, or `customer_finance` the value must be one of the following: `add_update`, `delete`, or `custom`.|

Reviewer: ```suggestion
| `behavior`                          | The action to perform | string | Required. For `advanced_pricing`, `catalog_product`, `customer_composite`, `requisition_list`, or `stock_sources`, the value must be: `append`, `replace`, or `delete`. For `customer`, `customer_address`, or `customer_finance`, you must use `add_update`, `delete`, or `custom`.|
```

`delivery_preferences` and `interests` are multiple select attributes for addresses and customers. They are represented as an array of strings.

**Requisition list payload:**

<Edition name="saas" />

> Each row in the `items` array represents an item in one requisition list. Rows are grouped by `(customer_email + list_name)` to form a parent-child relationship.

Reviewer: ```suggestion
Each row in the `items` array represents one requisition list item. Rows are grouped by `(customer_email + list_name)` to form a parent-child relationship.
```

The following table describes the possible contents of a requisition list item.

| Column | Required | Description |
|--------|----------|-------------|

> | `customer_email` | Yes | Customer email address (resolved to `customer_id` at import time) |

Reviewer: ```suggestion
| `customer_email` | Yes | Customer email address (resolved to `customer_id` on import) |
```
